lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -3728,7 +3728,12 @@ class StmtEmitter : public EmitterBase,
   LogicalResult visitSV(AliasOp op);
   LogicalResult visitSV(InterfaceInstanceOp op);
   LogicalResult visitStmt(OutputOp op);
+
```

Comment:
Nit: Unrelated whitespace change.

Reply:
It is related, I want to group the calls and the helper they resort to.